.netlify/state.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
@praveenweb Is this file required?

Reply:
It's Netlify deployment of `graphiql-online`. Removed now.